### ISSUE TITLE
anacoda 0.1.10 can's start anacoda server

### ISSUE THREAD

**Issue by wqhhust** [proofit404/anaconda-mode#284]
[0;31m[0m
I am using ubuntu 18.04, python 2.7.15rc1

➜  0.1.10 git:(develop) pwd
/home/oracle/.emacs.d/.local/etc/anaconda/0.1.10
➜  0.1.10 git:(develop) ls -l
total 20
drwxr-xr-x 4 oracle oracle 4096 May 16 05:15 anaconda_mode-0.1.10-py2.7.egg
-rw-r--r-- 1 oracle oracle  115 May 15 01:25 easy-install.pth
drwxr-xr-x 4 oracle oracle 4096 May 15 01:25 jedi-0.12.0-py2.7.egg
drwxr-xr-x 4 oracle oracle 4096 May 15 01:25 parso-0.2.0-py2.7.egg
drwxr-xr-x 4 oracle oracle 4096 May 15 01:25 service_factory-0.1.5-py2.7.egg

The anacoda buffer is:

[0;31mAssertionError[0mTraceback (most recent call last)
[0;32m<ipython-input-1-02efe3d5637d>[0m in [0;36m<module>[0;34m()[0m
[1;32m      3[0m [0msite[0m[0;34m.[0m[0maddsitedir[0m[0;34m([0m[0;34m'.'[0m[0;34m)[0m[0;34m[0m[0m
[1;32m      4[0m [0;32mimport[0m [0manaconda_mode[0m[0;34m[0m[0m
[0;32m----> 5[0;31m [0manaconda_mode[0m[0;34m.[0m[0mmain[0m[0;34m([0m[0msys[0m[0;34m.[0m[0margv[0m[0;34m[[0m[0;36m1[0m[0;34m:[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0m
[0m
[0;32m/home/oracle/.emacs.d/.local/etc/anaconda/0.1.10/anaconda_mode-0.1.10-py2.7.egg/anaconda_mode.pyc[0m in [0;36mmain[0;34m(args)[0m
[1;32m    106[0m [0;34m[0m[0m
[1;32m    107[0m [0;32mdef[0m [0mmain[0m[0;34m([0m[0margs[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0m
[0;32m--> 108[0;31m     [0;32massert[0m [0mlen[0m[0;34m([0m[0margs[0m[0;34m)[0m [0;34m==[0m [0;36m2[0m[0;34m,[0m [0margs[0m[0;34m[0m[0m
[0m[1;32m    109[0m     [0mhost[0m [0;34m=[0m [0margs[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m[0m[0m
[1;32m    110[0m     [0;32mif[0m [0margs[0m[0;34m[[0m[0;36m1[0m[0;34m][0m [0;34m!=[0m [0;34m""[0m[0;34m:[0m[0;34m[0m[0m

[0;31mAssertionError[0m: ['-c', "\nimport sys, site\nsite.addsitedir('.')\nimport anaconda_mode\nanaconda_mode.main(sys.argv[1:])\n", '127.0.0.1', '']

**Comment by proofit404** [comment]
Hi, did you upgrade elisp package?

**Comment by albertmichaelj** [comment]
I don't know if I'm running into the same issue, but I cannot start the anaconda server either on 0.1.10 on Windows 10. I'm getting the following error in the `*anaconda-mode*` buffer:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-7de66df3eaa2> in <module>()
      2 import sys, site
      3 site.addsitedir('.')
----> 4 import anaconda_mode
      5 anaconda_mode.main(sys.argv[1:])

c:\Users\Michael\.emacs.d\.cache\anaconda-mode\0.1.10\anaconda_mode-0.1.10-py3.6.egg\anaconda_mode.py in <module>()
     19 from functools import wraps
     20 
---> 21 from jedi import Script, create_environment
     22 from service_factory import service_factory
     23 

ImportError: cannot import name 'create_environment'
```

Please let me know if you have a suggestion on how to fix this. Thanks!

**Comment by kmkurn** [comment]
I had the same error and was able to fix it by upgrading jedi in my virtualenv to 0.12.

So, in my virtualenv, I have ipython installed which requires jedi. So, jedi 0.11 was installed in my virtualenv. Although in `~/.emacs.d/.cache/anaconda-mode/0.1.10` there is already `jedi-0.12.0-py3.6.egg`, somehow `anaconda-mode` still uses jedi 0.11 in my virtualenv. I verified this by running `from jedi import create_environment` and it gave me an `ImportError`. Thus, I upgraded jedi in my virtualenv to 0.12 and the error disappeared.

Is there a way so that `anaconda-mode` does not use jedi in our virtualenv?
